paramiko/sftp_client.py

```diff
@@ -692,7 +692,7 @@ def putfo(self, fl, remotepath, file_size=0, callback=None, confirm=True):
 
         The SFTP operations use pipelining for speed.
 
-        :param fl: opened file or file-like object to copy
+        :param fl object: opened file or file-like object to copy
```

Comment:
```suggestion
        :param object fl: opened file or file-like object to copy
```

Reply:
Thank you @jun66j5 for your code review.
You are correct: the type declaration should precede the argument name in the docstring.
This should be fixed now.